searchJob/templates/job_posts/job_post_list.html

```diff
@@ -28,36 +28,192 @@
+          <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-door-open" fill="currentColor"
+          xmlns="http://www.w3.org/2000/svg">
+
+
+          <path fill-rule="evenodd"
+          d="M1 15.5a.5.5 0 0 1 .5-.5h13a.5.5 0 0 1 0 1h-13a.5.5 0 0 1-.5-.5zM11.5
+          2H11V1h.5A1.5 1.5 0 0 1 13 2.5V15h-1V2.5a.5.5 0 0 0-.5-.5z"/>
+
+
+          <path fill-rule="evenodd"
+          d="M10.828.122A.5.5 0 0 1 11 .5V15h-1V1.077l-6
+          .857V15H3V1.5a.5.5 0 0 1 .43-.495l7-1a.5.5 0 0 1 .398.117z"/>
+
+          <path d="M8 9c0 .552.224 1 .5 1s.5-.448.5-1-.224-1-.5-1-.5.448-.5 1z"/>
+          </svg> My Account </button>
+
+          <button onclick="location.href='{% url 'logout' %}'" class="btn btn-sm btn-outline-secondary"
+          type="button"><svg width="1em" height="1em" viewBox="0 0 16 16"
+          class="bi bi-person-plus-fill" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" d="M1 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H1zm5-6a3 3 0 1 0 0-6 3 3 0 0 0
+           0 6zm7.5-3a.5.5 0 0 1 .5.5V7h1.5a.5.5 0 0 1 0 1H14v1.5a.5.5 0 0 1-1
+           0V8h-1.5a.5.5 0 0 1 0-1H13V5.5a.5.5 0 0 1 .5-.5z"/>
+          </svg> Log out </button>
+
+		{% else %}
+          <button onclick="location.href='{% url 'login' %}'" class="btn btn-sm btn-outline-secondary" type="button">
+          <svg width="1em" height="1em" viewBox="0 0 16 16" class="bi bi-door-open" fill="currentColor"
+          xmlns="http://www.w3.org/2000/svg">
+
+
+          <path fill-rule="evenodd"
+          d="M1 15.5a.5.5 0 0 1 .5-.5h13a.5.5 0 0 1 0 1h-13a.5.5 0 0 1-.5-.5zM11.5
+          2H11V1h.5A1.5 1.5 0 0 1 13 2.5V15h-1V2.5a.5.5 0 0 0-.5-.5z"/>
+
+
+          <path fill-rule="evenodd"
+          d="M10.828.122A.5.5 0 0 1 11 .5V15h-1V1.077l-6
+          .857V15H3V1.5a.5.5 0 0 1 .43-.495l7-1a.5.5 0 0 1 .398.117z"/>
+
+          <path d="M8 9c0 .552.224 1 .5 1s.5-.448.5-1-.224-1-.5-1-.5.448-.5 1z"/>
+          </svg> Sign In </button>
+
+          <button onclick="location.href='{% url 'accounts:signup' %}'" class="btn btn-sm btn-outline-secondary"
+          type="button"><svg width="1em" height="1em" viewBox="0 0 16 16"
+          class="bi bi-person-plus-fill" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+          <path fill-rule="evenodd" d="M1 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H1zm5-6a3 3 0 1 0 0-6 3 3 0 0 0
+           0 6zm7.5-3a.5.5 0 0 1 .5.5V7h1.5a.5.5 0 0 1 0 1H14v1.5a.5.5 0 0 1-1
+           0V8h-1.5a.5.5 0 0 1 0-1H13V5.5a.5.5 0 0 1 .5-.5z"/>
+          </svg>Sign Up</button>
+        {% endif %}
+    </div>
+    <h1> Available jobs  </h1>
+  
+  
 	{% if job_posts == "no relevant jobs for you" %}
-        <p>no relevant jobs for you<p>
+	<div class="no_job">
+        <p>Unfortunately there are no results matching your search..<br> Let’s try again</p>
+		<button class="search-btn" type="submit" onclick="history.back()" ><i class="fas fa-search"></i>
```

Comment:
Fix the indentation (If Im not wrong need to like line 114)

Reply:
I don't know why, but all the indentations look different in my local repo

for example:

PyCharm
![image](https://user-images.githubusercontent.com/69304551/103784788-0c0ad700-5043-11eb-95da-984c7fe52777.png)

notepad++

![image](https://user-images.githubusercontent.com/69304551/103778293-b5010400-503a-11eb-86f8-1c5a224db58b.png)

do you have any idea what is the reason? @bardielle

Reply:
I have no idea :( 
maybe you have spaces instead of tabs?